packages/clerk-js/src/ui/components/PricingTable/PricingTable.tsx

```diff
@@ -40,8 +37,13 @@ export const __experimental_PricingTable = (props: __experimental_PricingTablePr
       setDetailSubscription(activeSubscription);
       setShowSubscriptionDetailDrawer(true);
     } else {
-      setCheckoutPlan(plan);
-      setShowCheckout(true);
+      clerk.__internal_openCheckout({
+        planId: plan.id,
+        planPeriod,
+        orgId: subscriberType === 'org' ? organization?.id : undefined,
+        onSubscriptionComplete: onSubscriptionChange,
+        portalId: mode === 'modal' ? PROFILE_CARD_SCROLLBOX_ID : undefined,
```

Comment:
does `mode` indicate the context `PricingTable` is rendered in? I feel like I would expect `mode` to dictate how PricingTable behaves itself, but not necessarily how it behaves depending on where it's rendered.

As a semi-related nit, I might rename `mode` to `context`.

Reply:
Mode is set on the PricingTable and I think was originally adding following the other components, but there is no modal mode for PricingTable, so yeah, it is confusing, as its only a mounted component. Will refactor naming approach in a follow up PR.